src/schema/core.yaml

```diff
@@ -1677,27 +1674,51 @@ slots:
     description: the specific metabolite identifier
     range: ChemicalEntity
 
+  has_protein_identifications:
+    description: proteins that were annotated during a metaproteomics analysis and their associated count information
+    range: ProteinIdentification
+    multivalued: true
+    inlined_as_list: true
+
   all_proteins:
     description: the list of protein identifiers that are associated with the peptide sequence
     range: GeneProduct
     multivalued: true
+    deprecated: "not used. 2024-10.  See https://github.com/microbiomedata/nmdc-schema/issues/2157"
+
   best_protein:
     description: the specific protein identifier most correctly associated with the peptide sequence
     range: GeneProduct
+    deprecated: "not used. 2024-10.  See https://github.com/microbiomedata/nmdc-schema/issues/2157"
+
+  razor_protein:
+    description: the specific protein identifier identified in the metaproteomics analysis that is most correctly grouped to its associated peptide sequences
```

Comment:
Would "the metaproteomics analysis protein identifier derived from parsimonious peptide sequence association" work as a description?

Reply:
I will update language to indicate that these are determined using principle of parsimony